src/main/java/duke/exceptions/TaskIsCompleteException.java

```diff
@@ -0,0 +1,30 @@
+package duke.exceptions;
+
+/**
+ * This is a TaskIsCompleteException that extends DukeException.
+ * This exception is thrown when the user attempts to mark an already complete task as done.
+ */
+public class TaskIsCompleteException extends DukeException {
```

Comment:
hello really like how specific the exceptions are! but one thing is that you might want to change it to a noun form (i.e., CompletedTaskException) instead? 😄

Reply:
Thank you for the nice comments! I agree with everything you pointed out. Will work on it!